### DISCUSSION TITLE
Can't resize text and graphics, can't find info on how to...

### DISCUSSION THREAD

**Discussion by MarjaE2** [The-Wineskin-Project/WineskinServer#260] [Q&A]
Hi,

If I open apps in Wineskin, all the text is about 2 mm tall, too small for me to read. And some graphics are too small for me to see what I'm doing. The zoom option extends the window to cover the full screen, but doesn't resize anything in the window.

I've tried using "set screen options" and "virtual desktop" and reducing that resolution to 640x480, but the text remains as small as ever.

The Mac display options allow me to resize everything at once, but I have other fixes to fix fonts on some other apps. I end up with a mess where Wineskin text is still too small to read, other text is too big, and nothing fits on the screen. The Mac accessibility zoom options are worse.

Example, running nanDeck: ![Screen Shot 2022-05-13 at 12 03 17 AM](https://user-images.githubusercontent.com/12875862/168208992-8d250078-c791-4ac5-9fc1-03266a860218.png)

**Comment by MarjaE2** [comment]
I have a partial fix:

1. view package contents.
2. open Wineskin.
3. open the configuration editor.
4. find and click graphics, increase dpi to 144.

The fonts are now a reasonable size, but still too thin and spidery.

**Reply by Alexey-Danilchenko** [reply]
I have the same issue and the DPI increase works up until you exit wineskin. When .app bundle is then launched separately they are all gone. I cannot make them persist so any insights what is wrong would be helpful
